pyproject.toml

```diff
@@ -38,7 +38,7 @@ ecdsa = "*"
 bech32 = "*"
 requests = "*"
 protobuf = ">=4.21.6,<5.0dev"
-grpcio = "==1.53.2"
+grpcio = "*"
```

Comment:
I'm not quite sure about opening this as wide as this, but hopefully it won't haunt us in the future, when at some point the lastest grpcio package version causes things to break.

Reply:
This should remain fixed until we run `poetry lock` without the `--no-update` flag, and when we do, the tests will hopefully catch any breaking changes. I figured that pegging the version is only necessary when not doing so causes problems, so since it does not break anything at the moment, it seemed safe to get the latest version.